src/removeTask.test.js

```diff
@@ -1,10 +1,10 @@
-import removeTask from './removeTask';
-import Task from './task';
+import removeTask from './removeTask.js';
+import Task from './task.js';
 
 describe('removeTask function', () => {
   it('should remove a task from the tasks array', () => {
     const tasks = [new Task('Task 1', false, 1), new Task('Task 2', false, 2)];
-    const parentElement = { dataset: { index: 1 } }; // Simulate a DOM element
+    const parentElement = { dataset: { index: 1 } };
     const newTasks = removeTask(parentElement, tasks);
 
     expect(newTasks.length).toBe(tasks.length - 1);
```

Comment:
- [ ] Your tests only check for the remove and add methods. Kindly add tests that mock the DOM and/or local storage in both methods

  ![image](https://github.com/BHObserver/To-Do-List/assets/54848363/2dcbd761-8e89-47c3-9d78-ed712df9f2dd)

Reply:
@DammyShittu please could you pinpoint if this is a project requirement? I am asking this because the project requirement states that **Some** of those functions however will update localStorage and manipulate the DOM. **For those**, you will need to use mocks: This a condition in which our function might not be updating the local storage, or it needs to be implemented regardless. Please clarify further and help us to see this from the project requirements. Thank you.

Reply:
Hi @enocol sorry for the delay. Upon re-assessment, the requirement states that you should mock local storage and DOM for functions that mutate them. It appears your `removeTask` method mutates a `parentElement`. However, if the `parentElement` is merely a generic parameter that can be anything, and not really a DOM element then you're good to go.